app/src/main/java/co/runloop/influencer/adapter/ContactsAdapter.java

```diff
@@ -0,0 +1,88 @@
+package co.runloop.influencer.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import co.runloop.influencer.R;
+import co.runloop.influencer.model.Contact;
+
+public class ContactsAdapter extends RecyclerView.Adapter<ContactsAdapter.ContactViewHolder> {
+
+    private static final String TAG = ContactsAdapter.class.getSimpleName();
+
+    private List<Contact> contacts;
+    private LayoutInflater layoutInflater;
+    private Context context;
+
+    public ContactsAdapter() {
+        contacts = new ArrayList<>();
+    }
+
+    @Override
+    public void onAttachedToRecyclerView(@NonNull RecyclerView recyclerView) {
+        super.onAttachedToRecyclerView(recyclerView);
+        context = recyclerView.getContext();
```

Comment:
You right, Context can cause memory leak, as an option, i can set it to null, in on onDetachedFromRecyclerView()

Why not just pass the recycler view's context to the layout inflater?
Do you think this is better solution ? I just don't know a cost of retrieve LayoutInflater, so i thought save it would be better

Reply:
Opps, I think i wrong understand last path, i thought you told about var with LayoutInflater, do you mean why i saving context ?

Reply:
I'm saving context to address all calls to one point, i think this makes code simple

Reply:
Well, setting to `null` in `onDetachedFromRecyclerView` would be a good start. 
I still wonder if it's good practice to save the `context` reference as a member of the class, since it could cause memory leaks.

Regarding saving the inflater, this is probably a similar idea. While saving it might make things simpler, I don't think we should be saving things as members if they can be obtained on demand. 

So, removing the `layoutInflater` and `context` members seems like the better approach to me. This would make the `onAttachedToRecyclerView` implementation redundant as well.